lib/src/method_call_handler.dart

```diff
@@ -25,4 +27,20 @@ class AblyMethodCallHandler{
     return callbackResponse;
   }
 
+  bool realtimeAuthInProgress = false;
+  onRealtimeAuthCallback(AblyMessage message) async {
```

Comment:
Public members should have proper return types.

Reply:
Done, handled